ast/fodder.go

```diff
@@ -66,10 +66,10 @@ const (
 
 // FodderElement is a single piece of fodder.
 type FodderElement struct {
+	Comment []string
```

Comment:
Will this really affect performance? all 4 of these are 64 bits, not?

Reply:
Thanks for looking over this @sparkprime, it might vary slightly bases on architecture. I think a slice of strings is larger than 8 bytes, on my machine it's 24. I'm guessing it's two ints for the `len` and `cap`. Then a pointer for the backing array.

To answer you're question: if it will affect performance. Perhaps not significantly. Running the benchmarks locally only reported slight improvements.

But given that this change also enabled [`fieldalignment`](https://pkg.go.dev/golang.org/x/tools/go/analysis/passes/fieldalignment) on this package, it was easy enough to go through an fix up the spots it reported the structs as misaligned.